plugin/trino-clickhouse/src/main/java/io/trino/plugin/clickhouse/ClickHouseClient.java

```diff
@@ -241,6 +243,11 @@ public ClickHouseClient(
         JdbcTypeHandle bigintTypeHandle = new JdbcTypeHandle(Types.BIGINT, Optional.of("bigint"), Optional.empty(), Optional.empty(), Optional.empty(), Optional.empty());
         this.connectorExpressionRewriter = JdbcConnectorExpressionRewriterBuilder.newBuilder()
                 .addStandardRules(this::quoted)
+                .map("$equal(left: varchar, right: varchar)").to("left = right")
+                .add(new RewriteIn())
+                .map("$like(value: varchar, pattern: varchar): boolean").to("value LIKE pattern")
```

Comment:
For like with escape pattern ?

Reply:
Clickhouse does not support `like with escape`